front/app/containers/Admin/projects/components/ProjectRow/ProjectMoreActionsMenu.tsx

```diff
@@ -91,11 +90,17 @@ const ProjectMoreActionsMenu = ({
             window.confirm(formatMessage(messages.deleteProjectConfirmation))
           ) {
             setLoadingState('deleting', true);
-            await handleCallbackError(
-              () => deleteProject(projectId),
-              formatMessage(messages.deleteProjectError)
-            );
-            setLoadingState('deleting', false);
+
+            deleteProject(projectId, {
+              onSuccess: () => {
+                setError(null);
```

Comment:
Here as well, perhaps we don't need to manually manage error and loading  states.